ads/adunity.md

```diff
@@ -0,0 +1,33 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AdUnity
+
+## Example
+
+```html
+<amp-ad 
+    type="adunity"
+    width="300" height="250"
+    data-au-account="xxx" data-au-site="xxx" data-au-section="" data-au-zone="xxx"
+    data-au-dual="true">
+    <div placeholder></div>
```

Comment:
These `(placeholder, fallback)` are not specific to this ad and should be removed.